editor/file_system/dependency_editor.cpp

```diff
@@ -233,11 +349,21 @@ void DependencyEditor::_update_list() {
 		}
 
 		const String type = dep.contains("::") ? dep.get_slice("::", 1) : "Resource";
+		String name = path.get_file();
 		Ref<Texture2D> icon = EditorNode::get_singleton()->get_class_icon(type);
-		item->set_icon(0, icon);
-		item->set_metadata(0, type);
+		item->set_icon((int)DependencyEditorColumn::TYPE, icon);
+		item->set_text((int)DependencyEditorColumn::TYPE, type);
+		item->set_text((int)DependencyEditorColumn::NAME, name);
+		item->set_metadata((int)DependencyEditorColumn::TYPE, type);
+		item->set_text((int)DependencyEditorColumn::PATH, path);
+
+		if (!FileAccess::exists(path)) {
```

Comment:
This is already checked above. No need to perform it twice, just move this code to the previous condition.